src/utils/filterByPrice.js

```diff
@@ -0,0 +1,5 @@
+const filterByPrice = (genreFilteredBooks, maxPriceValue) => {
+    return genreFilteredBooks.filter(({ sellingPrice }) => sellingPrice <= maxPriceValue);
```

Comment:
no need for curly braces and return here

Reply:
Oh right. Thanks again